Orange/widgets/data/owsql.py

```diff
@@ -74,7 +74,9 @@ class Information(OWWidget.Information):
     class Error(OWWidget.Error):
         connection = Msg("{}")
         no_backends = Msg("Please install a backend to use this widget")
-        missing_extension = Msg("Database is missing extension{}: {}")
+
+    class Warning(OWWidget.Warning):
+        missing_extension = Msg("Database is missing extension: {}")
```

Comment:
The previous Error printed all missing extensions. I think that is better than just showing 1 (if 2 are missing), as it can be a pain to install that one and come back to Orange to find out later that another one is missing too.